src/dkg_hybridvss.erl

```diff
@@ -263,22 +234,32 @@ deserialize(#serialized_vss{id=Id,
          u2=erlang_pbc:binary_to_element(Element, SerializedU2),
          done=Done,
          session=Session,
-         sent_echo=SentEcho,
-         sent_ready=SentReady,
          received_commitment=ReceivedCommitment,
          readies=Readies,
-         commitment=dkg_commitment:deserialize(SerializedCommitment, Element)}.
+         commitments=maps:map(fun(_, V) -> dkg_commitment:deserialize(V, Element) end, SerializedCommitments)}.
 
 -spec status(vss()) -> map().
 status(VSS) ->
     #{id => VSS#vss.id,
       session => VSS#vss.session,
-      sent_echo => VSS#vss.sent_echo,
-      sent_ready => VSS#vss.sent_ready,
       received_commitment => VSS#vss.received_commitment,
-      num_echoes => dkg_commitment:num_echoes(VSS#vss.commitment),
-      num_readies => dkg_commitment:num_readies(VSS#vss.commitment),
-      echoes => maps:keys(dkg_commitment:echoes(VSS#vss.commitment)),
-      readies => maps:keys(dkg_commitment:readies(VSS#vss.commitment)),
+      %num_echoes => dkg_commitment:num_echoes(VSS#vss.commitment),
```

Comment:
Was this intentionally left commented out?

Reply:
They need to be repaired before this refactor is done.